frontend/src/layouts/BaseLayout.vue

```diff
@@ -148,13 +155,6 @@
               </base-tooltip>
             </span>
           </p>
-
-          <p v-if="isAccountSetup">
-            <q-icon class="dark-toggle icon" name="fas fa-user q-mr-xs" />
-            <span class="dark-toggle text-caption q-ml-sm">
-              Repeat <router-link class="hyperlink" :to="{ name: 'setup' }">account setup</router-link>
```

Comment:
Now that the stealth key registration is much simpler and you can no longer configure multiple ENS names, there's no reason to repeat account setup once completed, so I removed this from the footer